linera-rpc/src/grpc/conversions.rs

```diff
@@ -903,6 +907,13 @@ pub mod tests {
                 hash: CryptoHash::new(&Foo("".into())),
             },
         };
-        round_trip_check::<_, api::Notification>(notification);
+        let message = api::Notification::try_from(notification.clone()).unwrap();
```

Comment:
Also not a blocker, but this feels suspicious. It feels like we'd like to keep the `round_trip_check` for both cases, but split the new `ack` test into its own `test_empty_notification` test.

Seems okay for now, but we should revisit this test later.

Reply:
`round_trip_check` doesn't apply any more.